flake.nix

```diff
@@ -96,7 +99,14 @@
           inherit system;
           packages = packagesProf;
         };
-      })) // {
+
+        # Export ciJobs for tullia to parse
+        ciJobs = self.hydraJobs {
```

Comment:
This will  not support the [marlowe-cardano-integration jobset](https://github.com/input-output-hk/ci-ops/blob/4581b338c7937f300f3953d11152be6f04b8b98b/jobsets/default.nix#L320-L326), which overrides `plutus-apps`